### PR TITLE
FIX .apib document to make them work with Fabre for FIWARE R5 release

### PR DESCRIPTION
**Pull request by fgalan**
@FiwareULPGC 
@jmcanterafonseca

### CODE REVIEW

**Comment by fgalan** [comment]
I have tested both .apib document with fabre and apiary client and all renders seems to be generated ok.

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-TITLE: FIWARE-NGSI v2 Specification
-DATE: 30 September 2015
-VERSION: 2.0-latest
-PREVIOUS_VERSION: <placeholder>
-APIARY_PROJECT: fiware-ngsi-v2
+TITLE: FIWARE-NGSI v2 Specification (WIP)
+DATE: to be fixed upon next RC production
+VERSION: latest
+PREVIOUS_VERSION: stable
+APIARY_PROJECT: orioncontextbroker
```
Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: is this the real apiary project we have for NGSIv2?

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: It corresponds to:

http://docs.orioncontextbroker.apiary.io

Note that the project previously to this PR

http://docs.fiware-ngsi-v2.apiary.io

doesn't exist.

Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: cool, then if Fabre properly generates the link, LGTM

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 2]
```diff
@@ -47,7 +47,7 @@ In addition, note that a list of currently open discussions is available at
 
 Copyright (c) 2011-2016 Telecom Italia, Telefónica I+D and NEC. 
 
-## Legal Notice
+## License
 
 This specification is licensed under the
 [FIWARE Open Specification License (implicit patent license)]
```
Comment (Lenijas) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: There is a render problem related to:

```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```
FABRE cannot detect links if you insert a line break between `]` and `(`. So, the previous text should be something like (please note that there is no like break between `]` and `(` ):
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```

If you need to break the line, please place it in another position. You can even use multiplies break lines: 
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit 
patent license)](https://forge.fiware.org/plugins/
mediawiki/wiki/fiware/index.php/Implicit_Patents_License).

```
You can use http://dillinger.io/ to test different places where a line break can be inserted inside a link.

Comment (FiwareULPGC) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: There is a render problem related to:

```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```
FABRE cannot detect links if you insert a line break between `]` and `(`. So, the previous text should be something like (please note that there is no like break between `]` and `(` ):
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit patent license)]
(https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
```

If you need to break the line, please place it in another position. You can even use multiplies break lines: 
```
## License

This specification is licensed under the
[FIWARE Open Specification License (implicit 
patent license)](https://forge.fiware.org/plugins/
mediawiki/wiki/fiware/index.php/Implicit_Patents_License).

```
You can use http://dillinger.io/ to test different places where a line break can be inserted inside a link.

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Fixed (I hope) with 2da293c

**Comment by FiwareULPGC** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**